src/identifiers.md

```diff
@@ -60,4 +60,27 @@ See the [HIR chapter][hir-map] for more detailed information.
 
 ## In the MIR
 
-**TODO**
+- [`BasicBlock`] identifies a *basic block*. It points to an instance of
+  [`BasicBlockData`].
+- [`Local`] identifies a local variable in a function. Its associated data is in
+  [`LocalDecl`].
+- [`Field`] identifies a struct's, union's, or enum variant's field. It is used
+  as a "projection" in [`Place`].
+- [`SourceScope`] identifies a name scope in the original source code.
+- [`Promoted`] identifies a promoted constant (related to const evaluation).
+- [`GlobalId`] identifies a global variable: a `const`, a `static`, a certain
+  kind of `const fn`, or a promoted constant.
```

Comment:
I don't know much about these four; I was just trying to guess what they do based on the limit docs and looking through the source code. So let me know if I got it wrong!

Reply:
What does 'certain kind of const fn' mean?

Reply:
Quoting the docs for `GlobalId`:

> Uniquely identifies one of the following:
> - A constant
> - A static
> - A const fn where all arguments (if any) are zero-sized types

I didn't want to make the docs here too long and I was trying to limit the amount of duplication, but it probably makes sense to reword this.